### ISSUE TITLE
Add Levels with Enemy Bases in addition to normal spawn timers

### ISSUE THREAD

**Issue by 1RedOne** [Anuken/Mindustry-Suggestions#3253]
### Describe the content or mechanics you are proposing.

It would be fun if some levels included enemy stages, which would have fully developed enemy supply lines and factories instead of simply enemies dropping into an area.  

### Describe how you think this content will improve the game. If you're proposing new content, mention how it may add more gameplay options or how it will fill a new niche.

This would turn the play style on its head, with the player now in the role of invading enemy bases.  Enemies could use similar units to player units, with anti air arrays, long range salvos, and usage of smart end game units resources including high-end power generation.  

The player would be able to decide where to attack an enemy supply line for most damage, like attempting a risky and expensive attack on an enemy coolant line to take out power generating capability.  

Of course, the CPU player would still be able to call in routine drops of units in addition to those generated by their factories.

This would be fun because it allows the dev team to demonstrate a max efficiency enemy base and show how the gameplay elements allow the player to exploit and attack an enemy base.

Simply put: having an enemy base to attack sounds really fun!

### Before making this issue, check the boxes below to confirm that you have acknowledged them.

- [X] I have checked the [Trello](https://trello.com/b/aE2tcUwF/mindustry-trello) to make sure my suggestion isn't planned or implemented in a development version.
- [X] I am familiar with all the content already in the game or have glanced at the wiki to make sure my suggestion doesn't exist in the game yet.
- [X] I have read [README.md](https://github.com/Anuken/Mindustry-Suggestions/blob/master/README.md) to make sure my idea is not listed under the "A few things you shouldn't suggest" category.

**Comment by end-me-please** [comment]
"I am familiar with all the content already in the game or have glanced at the wiki to make sure my suggestion doesn't exist in the game yet."

the things you mentioned already existed back in v5..

**Comment by TheRealIvyX** [comment]
> I am familiar with all the content already in the game or have glanced at the wiki to make sure my suggestion doesn't exist in the game yet.

yep. totally.

**Comment by SMOLKEYS** [comment]
> Levels

Oh yeah, its small brain time for @1RedOne.

**Comment by random-guy1285** [comment]
My man didn't even hold out to fungal pass till complaining in the github

**Comment by SwitchLikesMlog** [comment]
this has to be a troll....
